src/pkg/api.go

```diff
@@ -41,6 +42,7 @@ func newGraphClient() *opslevel.Client {
 		opslevel.SetURL(apiURL),
 		opslevel.SetUserAgentExtra(userAgent),
 		opslevel.SetMaxRetries(5),
+		opslevel.SetTimeout(60*time.Second),
```

Comment:
This is the main change - everything else is whats needed to upgrade to opslevel-go 2023